packages/build/tamagui-build.js

```diff
@@ -622,10 +623,11 @@ async function esbuildWriteIfChanged(
                   //   ? null
                   //   : require.resolve('./babel-plugin-process-env-to-meta'),
                 ].filter(Boolean),
-              }).code
+              })
 
           // output to mjs fully specified
           await fs.writeFile(mjsOutPath, output, 'utf8')
+          await fs.writeFile(mjsOutPath + '.map', JSON.stringify(sourceMap), 'utf8')
```

Comment:
Also write the source map of `.mjs` files to disk.

Reply:
This seems to work magically, I think it's because in the `outString` (which is the input of `transform`) has a comment `//# sourceMappingURL=dev.js.map` and Babel `transform` is smart enough to read the original `dev.js.map` for example and generate a source map based on that.

This is just a guess, well at least if we remove the `//# sourceMappingURL=...` from `outString` (such as `outString = outString.split('\n').filter(s => !s.startsWith('//# sourceMappingURL')).join('\n')`) things will not work correctly.